dcplib/checksumming_io/checksumming_buffered_reader.py

```diff
@@ -1,17 +1,25 @@
 import hashlib
-import crcmod
+
 from io import BufferedReader
+
+import crcmod
+
 from .s3_etag import S3Etag
 
 
 class ChecksummingBufferedReader:
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, file_handler, read_chunk_size, *args, **kwargs):
+        """
+        :param read_file_size: optional file size for correctly setting the s3 etag chunk size;
+            defaults to S3Etag default if None
+        :param file_handler: the file handler to read from
```

Comment:
reorder this above the docblock for `read_file_size`

Reply:
✅